blog/2024-09-21-reflex-v0.6.0.md

```diff
@@ -0,0 +1,128 @@
+---
+author: Alek Petuskey
+date: 2024-09-21
+title: Reflex v0.6.0
+description: New features and improvements in Reflex v0.6.0
+image: /blog/reflex-060.webp
+meta: [
+    {"name": "keywords", "content": ""},
+]
+---
+
+```python exec
+from pcweb import constants
+import reflex as rx
+from pcweb.pages.docs import styling
+
+data = [
+    {"name": "A", "uv": 10, "pv": 110, "amt": 210},
+    {"name": "B", "uv": 20, "pv": 120, "amt": 230},
+    {"name": "C", "uv": 30, "pv": 120, "amt": 240},
+    {"name": "D", "uv": 30, "pv": 130, "amt": 210},
+    {"name": "E", "uv": 20, "pv": 140, "amt": 230},
+    {"name": "F", "uv": 40, "pv": 170, "amt": 250},
+    {"name": "G", "uv": 50, "pv": 190, "amt": 260},
+]
+```
+
+Reflex v0.6.0 is here! In this blog post, we’ll go over all the major improvements and changes since v0.5.0.
+
+### Decentralized Var Operations
+
+In this release, we reworked vars and made var operations to be more extensible. These changes make vars more stable and easier to work with along with adding more type safety.
+
+
+To an end reflex users, there is no visible change and it should be backward compatible with previous versions.
```

Comment:
We should link to the docs where we explain that Var.create makes a python value into a js literal. And Var() creates a value that renders directly in the generated JS. Noting that is_local and is_string are deprecated now.